### ISSUE TITLE
Design suggestions

### ISSUE THREAD

**Issue by AmrDeveloper** [AmrDeveloper/LinkHub#17]
In this thread everyone can share new design ideas and we can make a conversations them so we can improve the app design :D

**Comment by uli-on** [comment]
Hi Amr, here's something I derived from the existing oage layout. I kept all elements and measures, I simply removed redundancies, repetitions and most frame borders. I kept borders for folders, though, but removed the folder icons: the frames on folders symbolise their container function.

I also replaced serif with sans serif fonts for a more modern and system wide consistent design. You'll quickly find out what the current Android system font is, I'd not even have it on my computer, I simply used old standard Helvetica for mocking up a sans serif font look.

Before

![linkhub-ORI](https://user-images.githubusercontent.com/35634829/197362343-337569bc-3a1e-48d4-8c45-71379028dbca.jpg)


After

![linkhub-framed-folders](https://user-images.githubusercontent.com/35634829/197362355-75560896-57fc-446b-9f8b-8e8e846b56bf.jpg)

**Comment by AmrDeveloper** [comment]
Thanks @uli-on for your design it much simpler, I will keep this issue opened so we can discuss about more designs and review our designs

**Comment by mecuricat** [comment]
#### Design proposal for back button and new Save and Delete buttons.

The Delete button ❌ is kind of non-intuitive because it is usually associated with closing a window. In fact, I accidentally deleted a folder thinking it was the back button. I propose new buttons for Save, Delete and additionally a Back button.

<img src="https://user-images.githubusercontent.com/112750975/217211250-8f072b02-a079-429e-945d-3b99ae724d26.jpg" height="300px"/>

<img src="https://user-images.githubusercontent.com/112750975/217210879-24432b43-6e96-472a-ae52-877eac0d0097.jpg" height="300px"/>

**Comment by AmrDeveloper** [comment]
Thank @mecuricat, you are right about the problem between close/delete icon I will change them in next release

**Comment by mecuricat** [comment]
#### Design proposal inside a folder.

Let's say, I open my **Linux** folder. As it opens, you find that there is no way to telling which folder I am in. It just says "Links: 1".

<img src="https://user-images.githubusercontent.com/112750975/217226396-d58858eb-c28e-4f09-b584-8da6e1e30cf8.jpg" height="300px"/>

Also, there is no Create FAB inside folders. To create a link under **Linux** folder, I would have to go back to main page.

I propose the following:

1. Add the Create FAB inside each folder.
2. Display the folder icon and name, i.e. replace `Links:<count>` by `<folder icon> <folder name>: <count>`
3. Settings button next to Search button.

<img src="https://user-images.githubusercontent.com/112750975/217640483-1bcdee01-1309-42af-9727-02c859764edc.jpg" height="300px"/>

**Comment by SIRSteiner** [comment]
I would also prefer a modern sans serif font.

**Comment by mecuricat** [comment]
#### Proposal for paste buttons next to text fields:
Please add Paste buttons in each text field as illustrated in the image below.

<img src="https://user-images.githubusercontent.com/112750975/219933556-0ae8f467-a385-4a68-9e55-65e9e1dd0e55.jpg" height="300px"/>

Clicking on the Paste button will paste any text that is in the clipboard. Please let me know what you think.

**Comment by mecuricat** [comment]
#### Proposal for different themes:
In addition to Light and Dark themes, please implement Black (AMOLED), Material You and Custom.

In custom theme, like Simple Contacts Pro (v6.21.2 or lower) or Vinyl Music Player, you can adopt a Primary color and a Background color for theming different components.

<img src="https://user-images.githubusercontent.com/112750975/217638160-1954dbd8-43c2-4271-9921-1b03cb60c895.jpg" height="300px"/>

**Comment by SIRSteiner** [comment]
@AmrDeveloper Do you working on themes at the moment? If not, shall I begin with moving all colors to theme and night/theme?

**Comment by AmrDeveloper** [comment]
> @AmrDeveloper Do you working on themes at the moment? If not, shall I begin with moving all colors to theme and night/theme?

We need to solve the issues and release then we can redesign the UI To be modern and simple and use some idea from this discussion then we can work on themes and colors :D

**Comment by mecuricat** [comment]
#### Proposal for uniform colors:
The red and blue colors are very bright and feel out of place in an otherwise black background. Please color the Save and Delete buttons same as the text color of **LinkHub** in the image below.

![color](https://user-images.githubusercontent.com/112750975/219933000-7f8b370c-d79e-468f-9ce1-a4a9e554638b.jpg)

This will provide a uniform look. Please let me know what you think.

**Comment by SIRSteiner** [comment]
Yes, I think that could be better especially if we don't need the save button.

@AmrDeveloper Is it time to move all colors to theme?

**Comment by AmrDeveloper** [comment]
@SIRSteiner Sure we will but first, we need to make sure all important issues are solved, then we will move colors to theme and we need to think about some features and how we can make them clear for example in remember last folder request should we mark this folder to use as last used or make it the default for next folders? or maybe a hint 😄,

**Comment by bovergaauw** [comment]
May I suggest that links are sortable manually and permanently retain the manual sort order?

**Comment by bovergaauw** [comment]
May I suggest that slide selectors (e.g. the one for pinning a link) become grey when off?

**Comment by AmrDeveloper** [comment]
> May I suggest that links are sortable manually and permanently retain the manual sort order?

This is a good idea we can add many options for sorting and The current sorting can be named sorting by usage

**Comment by AmrDeveloper** [comment]
> May I suggest that slide selectors (e.g. the one for pinning a link) become grey when off?

Thanks for your suggestion I will check it

**Comment by backtabber** [comment]
I also vote for a san serif font, and removing the word "Link" from the end of the link description.

**Comment by feelsavvy** [comment]
Make this app Meterial You Design
